src/main/component/Editor.svelte

```diff
@@ -55,13 +61,15 @@
   import { createEventDispatcher, onDestroy, onMount } from 'svelte';
   import type { TinyMCE, Editor as TinyMCEEditor } from 'tinymce';
   type EditorOptions = Parameters<TinyMCE['init']>[0];
+  type Version = `${'4' | '5' | '6' | '7'}${'' | '-dev' | '-testing' | `.${number}` | `.${number}.${number}`}`;
 
   import { bindHandlers } from './Utils';
   export let id: string = uuid('tinymce-svelte'); // default values
   export let inline: boolean | undefined = undefined;
   export let disabled: boolean = false;
   export let apiKey: string = 'no-api-key';
-  export let channel: string = '6';
+  export let licenseKey: string | undefined = undefined;
+  export let channel: Version = '7';
```

Comment:
I feel like the Version type is a bit overkill. String should be good enough, and if we really want to keep the type maybe `Channel` would make more sense?

Reply:
```suggestion
  export let channel: Channel = '7';
```